static/js/brand-store/types/shared.ts

```diff
@@ -3,13 +3,44 @@ export type RouteParams = {
 };
 
 export type Member = {
-  current_user: {};
-};
-
-export type CurrentMember = {
   displayname: string;
   email: string;
   id: string;
   roles: string[];
   username: string;
+  current_user: boolean;
+};
+
+export type BrandStores = {
+  brandStores: {
+    brandStoresList: Array<{}>;
```

Comment:
I'm guessing your plan is to flesh these objects out more in the future?

Reply:
Yes, this will most likely happen when I do the navigation which hopefully will be in the upcoming iteration